packages/api/amplify_api_dart/lib/src/graphql/web_socket/types/web_socket_event.dart

```diff
@@ -88,6 +88,19 @@ class NetworkLossEvent extends NetworkEvent {
   String get runtimeTypeName => 'NetworkLossEvent';
 }
 
+/// Discrete class for when the process is unpaused
+/// Triggers when AppLifecycleListener detects the process has been unpaused.
+class ProcessUnpausedEvent extends WebSocketEvent {
```

Comment:
nit: small rename to conform to lifecycle event title
```suggestion
class ProcessResumeEvent extends WebSocketEvent {
```

Reply:
renamed!